settings/pkg/config/defaults/defaultconfig.go

```diff
@@ -44,13 +46,22 @@ func DefaultConfig() *config.Config {
 			Addr:      "127.0.0.1:9191",
 			Namespace: "com.owncloud.api",
 		},
-		DataPath: path.Join(defaults.BaseDataPath(), "settings"),
+		StoreType: "metadata", // use metadata or filesystem
+		DataPath:  path.Join(defaults.BaseDataPath(), "settings"),
 		Asset: config.Asset{
 			Path: "",
 		},
 		TokenManager: config.TokenManager{
 			JWTSecret: "Pive-Fumkiu4",
 		},
+
+		Metadata: config.Metadata{
+			GatewayAddress:    "127.0.0.1:9142",
+			StorageAddress:    "127.0.0.1:9215",
+			ServiceUserID:     "ddc2004c-0977-11eb-9d3f-a793888cd0f8",
```

Comment:
should we really use the admin user here? why can't we use a system user like for the accounts service?

Reply:
Well. Seems like it still doesn't work :disappointed:

Reply:
@wkloucek I found a way around it. Can you check again?